build_tools/azure/posix.yml

```diff
@@ -73,6 +74,30 @@ jobs:
         testRunTitle: ${{ format('{0}-$(Agent.JobName)', parameters.name) }}
       displayName: 'Publish Test Results'
       condition: succeededOrFailed()
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.9'
+      displayName: Place Python into path to update issue tracker
+      condition: and(succeededOrFailed(), eq(variables['CREATE_ISSUE_ON_TRACKER'], 'true'),
+                     in(variables['Build.Reason'], 'Schedule', 'IndividualCI'))
```

Comment:
does this mean we run this for individual CI runs? I thought we don't want that, since we only have this for nightly scipy-dev runs